envs/linc/managed-jupyterhub.yaml

```diff
@@ -0,0 +1,356 @@
+              spawner.environment["GITHUB_EMAIL"] = auth_state["github_user"]["email"]
+      c.KubeSpawner.modify_pod_hook = modify_pod_hook  # noqa
+      c.JupyterHub.authenticator_class = IsDandiUserAuthenticator  # noqa
+      c.GitHubOAuthenticator.enable_auth_state = True  # noqa
+prePuller:
+  continuous:
+    enabled: false
+  hook:
+    enabled: false
+proxy:
+  https:
+    enabled: true
+    hosts:
+      - ${jupyterhub_domain}
+    type: offload
+  service:
+    annotations:
+      service.beta.kubernetes.io/aws-load-balancer-backend-protocol: tcp
+      service.beta.kubernetes.io/aws-load-balancer-connection-idle-timeout: '3600'
+      service.beta.kubernetes.io/aws-load-balancer-cross-zone-load-balancing-enabled: 'true'
+      service.beta.kubernetes.io/aws-load-balancer-ip-address-type: ipv4
+      service.beta.kubernetes.io/aws-load-balancer-nlb-target-type: ip
+      service.beta.kubernetes.io/aws-load-balancer-scheme: internet-facing
+      service.beta.kubernetes.io/aws-load-balancer-ssl-cert: ${ssl_cert_arn}
+      service.beta.kubernetes.io/aws-load-balancer-ssl-ports: https
+      service.beta.kubernetes.io/aws-load-balancer-type: external
+scheduling:
+  podPriority:
+    enabled: true
+  userPlaceholder:
+    enabled: false
+    replicas: 1
+  userPods:
+    nodeAffinity:
+      matchNodePurpose: require
+  userScheduler:
+    enabled: true
+singleuser:
+  allowPrivilegeEscalation: true
+  cmd: null
+  cpu:
+    guarantee: 0.5
+    limit: 12
+  defaultUrl: /lab
+  extraEnv:
+    CHOWN_EXTRA: /home/shared
+    CHOWN_HOME: 'yes'
+    CHOWN_HOME_OPTS: -R
+    GRANT_SUDO: 'yes'
+    NOTEBOOK_ARGS: --allow-root
+  extraPodConfig:
+    securityContext:
+      fsGroup: 100
+  fsGid: 0
+  initContainers:
+    - command:
+        - sh
+        - -c
+        - 'chmod 0775 /nfs; chown 1000:100 /nfs; chmod 0775 /shared; chown 1000:100
+          /shared; chmod 0555 /readonly
+
+          '
+      image: alpine
+      name: nfs-fixer
+      securityContext:
+        runAsUser: 0
+      volumeMounts:
+        - mountPath: /nfs
+          name: persistent-storage
+          subPath: home/{username}
+        - mountPath: /shared
+          name: persistent-storage
+          subPath: shared
+        - mountPath: /readonly
+          name: persistent-storage
+          subPath: readonly
+  lifecycleHooks:
+    postStart:
+      exec:
+        command:
+          - sh
+          - -c
+          - 'gitpuller https://github.com/dandi/example-notebooks master dandi-notebooks;
+            chown -R jovyan:users dandi-notebooks; datalad install https://github.com/dandi/dandisets;
+            /opt/conda/envs/allen/bin/python -m ipykernel install --user --name allen
+            --display-name "Python (Allen SDK)"; /opt/conda/bin/pip install --upgrade
+            dandi; git config --global user.email "$${GITHUB_EMAIL}"; git config --global
+            user.name "$${GITHUB_USER}"
```

Comment:
I am sure you are already planning to update this script, but heads up that this managed script needs to be updated.

Reply:
Sorry Kabi, could you clarify what you mean by "this managed script needs to be updated" -- I assume you mean that I'll need to redeploy?

Reply:
Hi Aaron, it looks like the `jupyter-overrides.yaml` and `managed-jupyterhub.yaml` are out of sync.  If I am not mistaken, once you run [install.sh](https://github.com/dandi/dandi-hub/blob/main/install.sh) for the deployment, this `managed-jupyterhub.yaml` file will be updated and then you will need to push those changes to this pull request.

Reply:
I think the only change is the tf state that is upload to S3 if I'm not mistaken, but @asmacdo would know with certainty

Reply:
The first part of the install script does update the managed jupyterhub file from the overrides. When making a PR like this I usually just comment out the deployment part and execute to get them aligned.

We should make that a cleaner process, (would be nice to add a check to ci also) but for now you do need to run that part of the script.

Reply:
Thanks for clarifying, Austin.  Thats an easy fix.

Reply:
@asmacdo let me know if you might have 5 minutes to zoom -- I _think_ I understand you correctly, but I want to make sure I make the proper steps when re-deploying

Reply:
Resolving this thread, as @asmacdo and I deployed successfully.